### ISSUE TITLE
Can not extract JP text after building into exe with Pyinstaller

### ISSUE THREAD

**Issue by gunrosen** [pdfminer/pdfminer.six#575]
Hi there.
Firstly, let's say pdfminer.six works properly when development.
It means it can run with .py 
![image](https://user-images.githubusercontent.com/3174484/106103399-79f27c00-6173-11eb-9221-316ef3db6a31.png)

After I use Pyinstaller to make production version. It does not extract text. LTRect, LTLine, LTCurve can recognize but LTText, LTContainer can't. 

> pyinstaller --onefile test_pdf.py

![image](https://user-images.githubusercontent.com/3174484/106103737-f9804b00-6173-11eb-9945-b7e66ff63e4a.png)
My environment
> Windows 10 Home Single Language Version 2004 
> Python 3.9.1
> Pyinstaller 4.2
> pdfminer.six==20201018
> six==1.15.0

Pyinstaller log: [pyinstaller.log](https://github.com/pdfminer/pdfminer.six/files/5885466/pyinstaller.log)
SamplePDF file: [ADEM_3.pdf](https://github.com/pdfminer/pdfminer.six/files/5885442/ADEM_3.pdf)
Sample source code: I print text into console
```
from pdfminer.high_level import extract_pages
from pdfminer.layout import LTTextContainer
from pdfminer.layout import LTRect
import sys

print("Read file %s" % sys.argv[1])
if __name__ == '__main__':
    # print("WE TRY TO TEST")
    for page_layout in extract_pages(sys.argv[1]):
        for element in page_layout:
            if isinstance(element, LTTextContainer):
                print(element.get_text())
            elif isinstance(element, LTRect):
                pass
```

I am not sure the issue happen on pdfminer.six or pyinstaller or maybe cause of file format. But I don't know how to find out the solution for this problem.
Please give me a help with this issue. 
Thank you.

**Comment by hide212131** [comment]
I ran into a similar problem. (In my case, [camelot](https://github.com/camelot-dev/camelot) also didn't work with pyinstaller. camelot uses pdfminer internally).

As I investigated, I had a feeling it was something around the “cmap”, so I copied cmap from the distribution of pdfminer with the following configuration, and was able to get the text!
(This is the case without option “—onefile” of pyinstaller.)

```
dist/
+-- xxx.exe
+-- pdfminer/
    +-- cmap/
        +--78-EUC-H.pickle.gz
        +--...
```

I hope this is helpful.
